public/js/todoAppController.js

```diff
@@ -0,0 +1,43 @@
+todoApp.controller('todoAppController', function($scope) {
+
+$scope.todos = [];
+
+  $scope.addTodo = function  () {
+    $scope.todos.push({taskName : $scope.newTodo , isComplete : false });
+  };
+
+  $scope.clearCompleted = function () {
+    for (var i = 0; i < $scope.todos.length; i++) {
```

Comment:
instead of the for loop you could use filter:
$scope.todos = $scope.todos.filter(function(task){
      return !task.isComplete
    })